src/dungeon.cc

```diff
@@ -4185,12 +4158,6 @@ static void process_command(void)
 			do_cmd_macro_recorder();
 			break;
 		}
-	case CMD_BLUNDER:
-		{
-			if (do_control_walk()) break;
-			do_cmd_walk(always_pickup, FALSE);
```

Comment:
I'll attempt that. It will probably break things all over the place, but we'll see.